modules/ztp-creating-the-siteconfig-custom-resources.adoc

```diff
@@ -83,6 +84,8 @@ spec:
                 next-hop-address: 2620:52:0:10e7::fc
                 table-id: 254
 ----
+<1> If you are using SecureBoot, add this line to prevent failures
```

Comment:
Should this say `UEFI SecureBoot`? (in backticks)?

Reply:
made this change.